vllm/config.py

```diff
@@ -349,9 +349,35 @@ def get_num_attention_heads(self,
         return num_heads // parallel_config.tensor_parallel_size
 
     def get_num_layers(self, parallel_config: "ParallelConfig") -> int:
-        total_num_hidden_layers = self.hf_text_config.num_hidden_layers
+        total_num_hidden_layers = getattr(self.hf_text_config,
```

Comment:
Yeah , tests failed because they didn't have num_hidden_layers in HF config , see https://github.com/vllm-project/vllm/blob/cd9ba35121abca25bf8a3f9c7cb2bb091746416e/vllm/config.py#L244 they also treat it there this way